en/firefox/products.ftl

```diff
@@ -4,63 +4,73 @@
 
 ### URL: https://www-dev.allizom.org/products/
 
-# Obsolete string
 # HTML page title
-firefox-products-firefox-is-more-than-a-browser = { -brand-name-firefox } is more than a browser
+firefox-products-mozilla-products = { -brand-name-mozilla }’s products
 
-# HTML page title
+# HTML page description
+firefox-products-mozillas-family-of-products = { -brand-name-mozilla }’s family of products are all designed to keep you safer and smarter online
+
+firefox-products-products = Products
+firefox-products-firefox-beginning-v2 = { -brand-name-firefox } is just the beginning.
+
+# Obsolete string (expires: 2024-07-03)
 firefox-products-firefox-beginning = { -brand-name-firefox } is just the beginning
 
-# Obsolete string
-# HTML page description
-firefox-products-its-a-whole-family-of-products = It’s a whole family of products designed to keep you safer and smarter online.
+firefox-products-mozillas-family-of-products-sentence = { -brand-name-mozilla }’s family of products are all designed to keep you safer and smarter online.
 
-firefox-products-mozillas-family-of-products = { -brand-name-mozilla }’s family of products are all designed to keep you safer and smarter online
+# appears after the name of a product with independant branding. Example: Fakespot by Mozilla
+firefox-products-by-mozilla = by { -brand-name-mozilla }
 
-# Obsolete string
-# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
-firefox-products-firefox-is-more-than-a-browser-emphasis = { -brand-name-firefox } is <strong>more</strong> than a browser
+## Firefox
 
-firefox-products-mozilla-monitor = { -brand-name-mozilla-monitor }
+firefox-products-firefox = { -brand-name-firefox }
+firefox-products-get-the-browsers-that-block = Get the browsers that block 2000+ data trackers automatically. Enhanced Tracking Protection comes standard in every { -brand-name-firefox } browser.
```

Comment:
Why "browsers"? That seems wrong, given it points to the download page for Firefox desktop release.
https://www-dev.allizom.org/en-US/products/